plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/types.ts

```diff
@@ -59,3 +59,5 @@ export enum StatusLevel {
 	Error = 'error',
 	Info = 'info',
 }
+
+export type MySubscriptionsTable = 'installed' | 'available';
```

Comment:
Not feeling super strong about this, but WDYT of doing this instead?

```
export enum MySubscriptionsTable {
  Installed = 0,
  Available = 1,
}
```

Might save a few bytes in the built JS (numeric IDs vs strings)

Reply:
WDYT is the benefit of using unions? Trying to understand if it's worth the cost for the (small) performance improvement?